docs/readme.md

```diff
@@ -1,21 +1,27 @@
 # AGAGD
 
 ## Overview
-An app to display users, games, tournaments, and ratings information on the AGA website.  
+An app to display users, games, tournaments, and ratings information on the AGA website.
 A port of the old eurogo Games Database to python, for the AGA
 
  - agagd_core/     --
- - agagd/          -- 
+ - agagd/          --
 
 ## Development
 
+### Code Style
+The AGAGD project uses pre-commit for some basic linting, so before it is required to have pre-commit installed and any errors fixed before making a pull request. ([Pre Commit](https://pre-commit.com/), [DJango Code Style](https://docs.djangoproject.com/en/3.2/internals/contributing/writing-code/coding-style/))
```

Comment:
Prose alternative:

The AGAGD project uses [pre-commit](https://pre-commit.com) for some basic linting and mostly follows [Black](https://github.com/psf/black).

Reply:
Sounds Better and more descriptive.

Reply:
Thanks.